middleware/remaps.js

```diff
@@ -0,0 +1,15 @@
+module.exports.remapDifficultyTexts = {
```

Comment:
This should belong in front end side of things helpers. You can just pass the difficulty integer to the ejs template and remap it there.

Reply:
Note for myself: helpers.js in public folder